modules/installation-rhv-building-ignition-files.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_rhv/installing-rhv-user-infra.adoc
+
+[id="installation-rhv-building-ignition-files_{context}"]
+= Building the ignition files
+
+To build the Ignition files from the manifest files you just generated and modified, you run the installation program. This action creates a {op-system-first} machine, `initramfs`, which fetches the ignition files and performs the configurations needed to bring a node to life.
+
+In addition to the Ignition files, the installation program generates the following:
+
+* An `auth` directory that contains the admin credentials for connecting to the cluster with the `oc` and `kubectl` utilities.
+* A `metadata.json` file that contains information such as the OCP cluster name, OCP cluster ID, and the `infraID` for the current installation.
+
+The Ansible playbooks for this installation process use the value of `infraID` as a prefix for the virtual machines they create. This prevents naming conflicts when there are multiple installations in the same oVirt/{rh-virtualization} cluster.
+
+[IMPORTANT]
+====
+Certificates in Ignition configuration files expire after 24 hours. Complete the cluster installation and keep the cluster running in a non-degraded state for 24 hours so that the first certificate rotation can finish.
```

Comment:
This comes from Roberto's instructions [at the bottom of this subsection](https://github.com/openshift/installer/blob/master/docs/user/ovirt/install_upi.md#ignition-configs). It looks like he probably ran into a problem because his cluser was degraded/not running when the first 24-hour cycle ended. He also uses the phrase "you must," so it seems important.